README.rst

```diff
@@ -585,6 +585,9 @@ The variables that can be set are:
 |jsdoc_private          | jsdoc should produce documentation for private       |
 |                       | entities. True by default.                           |
 +-----------------------+------------------------------------------------------+
+|no_jsdoc_private       | jsdoc should not produce documentation for private   |
+|                       | entities. False by default.                          |
++-----------------------+------------------------------------------------------+
```

Comment:
The documentation should mention the command line flag but should not intimate that it is a configuration option.